identity/service.go

```diff
@@ -351,9 +351,7 @@ func (s *Service) propagateDataHandler(msg network.Message) {
 		case *ProposeSend:
 			p := msg.(*ProposeSend)
 			sid.Proposed = p.Propose
-			if len(sid.Proposed.Votes) == 0 {
-				sid.Proposed.Votes = map[string]*crypto.SchnorrSig{}
-			}
+			sid.Proposed.Votes = make(map[string]*crypto.SchnorrSig)
```

Comment:
what is the reason you removed the `if` ? Do you *always* setup the votes by default when receiving a `ProposeSend` ?